client/src/styles/global.scss

```diff
@@ -84,19 +84,61 @@ $primary-color: #112f4e;
   }
 
   // this is the title
-  .usa-logo__text {
-    font-size: 1.8em;
-    font-family: "serif";
-  }
+  @include at-media("desktop") {
+    .j40-title {
+      font-size: 0.9em;
+      padding-top: 25px; // allows vertical centering on logo and text
```

Comment:
if the font size is responsive depending on root document size, but the padding is specific to pixels, this will break on some views. not worth fixing, but probably code smell

Reply:
Good call. Thanks!

Reply:
Ends up this was not needed.